### ISSUE TITLE
[Enhancement]: jungle village plz?

### ISSUE THREAD

**Issue by Hexagram2021** [Viola-Siemens/EmeraldCraftMod#2]
### Feature Type

- [ ] Loot Table for Blocks or Continuous Miner
- [ ] Texture for Blocks or Entities
- [ ] Biomes
- [ ] Recipes and GUI
- [ ] Professions and Trades of Villagers
- [X] Features
- [X] Structures
- [ ] Useful Blocks and Items
- [ ] Entities
- [X] Other

### Description of the feature

there is a jungle villager type in minecraft already but no jungle village generated in jungle biomes. i see the swamp village you made and i love it. would you like to make a jungle village in higher version?
i think the bamboo block and related blocks are suitable for building. i will be really thankful if you make chinese style buildings.

**Comment by Viola-Siemens** [comment]
yes, im planning to work on it even several versions ago. but im not satisfied about what i've designed so i decided to put aside. but now bamboo blocks are added, so i will try.
btw, are you chinese?

**Comment by Hexagram2021** [comment]
yes, and i enjoy playing your mods.

**Comment by Viola-Siemens** [comment]
得，都中国人用中文就行。
1.19.3我先试试能不能加上，不行就拖到1.20了。我这还有别的项目，不一定有时间，但我尽量。

**Comment by Hexagram2021** [comment]
草，好，谢谢作者

**Comment by Viola-Siemens** [comment]
> 得，都中国人用中文就行。 1.19.3我先试试能不能加上，不行就拖到1.20了。我这还有别的项目，不一定有时间，但我尽量。

I'll explain this in English. I'm going to add jungle village after 1.19.3 or 1.20 because bamboo blocks are added after this version. But I'm kind of busy right now so it might take longer.

**Comment by Viola-Siemens** [comment]
Here's a good example for submitting an issue.
